spyt-package/src/main/python/spyt/conf.py

```diff
@@ -3,19 +3,11 @@
 from spyt.dependency_utils import require_yt_client
 require_yt_client()
 
-from yt.wrapper import get, YPath, list as yt_list, exists  # noqa: E402
+from yt.wrapper import get, YPath, list as yt_list, exists, YtClient  # noqa: E402
```

Comment:
I think we can get rid of importing and using `get` method because now you use `YtClient` instead

Reply:
Inside the `SpytDistributions` – yes, but I am not sure about the external methods: `client` can still be `None` in those.